README.adoc

```diff
@@ -334,6 +333,10 @@ The `CYPRESS_RECORD_KEY` environment variable needs to be set for this to work.
 |===
 |Testcontainers-cypress |Testcontainers | Cypress
 
+|https://github.com/wimdeblauwe/testcontainers-cypress/releases/tag/testcontainers-cypress-1.9.0[1.9.0]
+|https://github.com/testcontainers/testcontainers-java/releases/tag/1.19.0[1.19.0]
+|https://docs.cypress.io/guides/references/changelog.html#13-2-0[13.2.0]
```

Comment:
There is now a 13.3.0 I noticed. Maybe update to that one https://docs.cypress.io/guides/references/changelog#13-3-0 ?

Reply:
Ofcourse!